packages/relay-compiler/core/GraphQLSchemaUtils.js

```diff
@@ -82,8 +82,6 @@ function canHaveSelections(type: GraphQLType): boolean {
  * Implements duck typing that checks whether a type has an id field of the ID
  * type. This is approximating what we can hopefully do with the __id proposal
  * a bit more cleanly.
- *
- * https://github.com/graphql/graphql-future/blob/master/01%20-%20__id.md
```

Comment:
This link still exist. As long as we have to comment above, it makes sense for me to keep?

Reply:
I don't really understand what you mean. If I open this link (or just [the repo of it](https://github.com/graphql/graphql-future/)) I get a 404-Github page. Is this repo private?
![image](https://user-images.githubusercontent.com/29959150/53980881-b812cc80-4111-11e9-88e2-2bac3fefb7dc.png)

Reply:
Oh, I guess it's been made private as the last change was 3 years ago.